rust-toolchain.toml

```diff
@@ -2,4 +2,4 @@
 channel = "1.80.0"
 targets = ["wasm32-unknown-unknown"]
 profile = "default"
-components = ["rls"]
+components = ["rls", "rust-analyzer"]
```

Comment:
This only applies to the Cargo build, right? Also I was under the impression that Cargo analyzer did not work (well enough) in our repo, is that now fixed?

Reply:
I've been using it with reasonable success, it just disappears every time we upgrade the toolchain until I remember to reinstall it 😅. I _think_ it's been enabled [here](https://github.com/dfinity/ic/blob/99c86c9e019de17e2633f7112a5dcc84a652d55d/.vscode/extensions.json) for VSCode users too?

I remember at one point using [this](https://github.com/dfinity/ic/blob/99c86c9e019de17e2633f7112a5dcc84a652d55d/.vscode/bazel-check-wrapper) bazel wrapper, but it doesn't look like that's supported anymore.

Reply:
> it's been enabled [here](https://github.com/dfinity/ic/blob/99c86c9e019de17e2633f7112a5dcc84a652d55d/.vscode/extensions.json) for VSCode users too?

This should never have been committed! I'll remove it